### ISSUE TITLE
some problems in inference code

### ISSUE THREAD

**Issue by pixelsmaker** [seoungwugoh/STM#13]
`B_list['o'].append( (torch.sum(masks[:,1:o], dim=1) + \
                torch.sum(masks[:,o+1:num_objects+1], dim=1)).clamp(0,1) )`

  I don't understand the meaning of B_list['o'], besides, 
` x = self.conv1(f) + self.conv1_m(m) + self.conv1_o(o) `
 f represents image, m means mask,  while o  I don't know its meaning. 

I would be grateful if you could answer my questions.

**Comment by seoungwugoh** [comment]
Hi, `o` stands for other objects' mask. It is used only if there are other objects else it is a zero mask.
`o` is created by summing all the masks except for the target object.

**Comment by pixelsmaker** [comment]
> Hi, `o` stands for other objects' mask. It is used only if there are other objects else it is a zero mask.
> `o` is created by summing all the masks except for the target object.!
thank you for you reply~
